src/Amadeus/Client/RequestOptions/Travel/Pax.php

```diff
@@ -50,4 +50,6 @@ class Pax extends LoadParamsFromArray
      * @var bool
      */
     public $passengerContactRefused;
+
+    public $paxRefId;
```

Comment:
@yustikodm could you please add PHPDoc as we have for other properties?
That would help other people to understand what type of data should be provided.